src/models/transactions/accountDelete.ts

```diff
@@ -0,0 +1,28 @@
+import { ValidationError } from "../../common/errors";
+import { BaseTransaction, verifyBaseTransaction } from "./common";
+
+export interface AccountDelete extends BaseTransaction {
+    TransactionType: "AccountDelete";
+    Destination: string;
```

Comment:
I think we can add `Fee: 5` to this typescript definition. Fee should always be 5 in `AccountDelete` Transactions IIRC.